python/pyspark/sql/context.py

```diff
@@ -578,6 +606,9 @@ def __init__(self, sqlContext):
     def register(self, name, f, returnType=StringType()):
         return self.sqlContext.registerFunction(name, f, returnType)
 
+    def registerUDF(self, name, f):
```

Comment:
Maybe we can add doc here by `registerUDF.__doc__ == SQLContext.registerUDF.__doc__` similar to doc for `register`.
(We should do it for `registerJavaFunction` and `registerJavaUDAF`, too?)

Reply:
Yup +1 like https://github.com/apache/spark/pull/20217/files/f25669a4b6c2298359df1b9083037468652cd141#r160861434 but how about checking and doing this in batch? Seems we should fix the doctests too.

Reply:
The examples in these docs are different. Thus, I prefer to keeping it untouched.